# * We need to load a external plugins from another binary compatible slot
# * mima-plugin isn't available for 0.10 yet

diff --git a/build.sc b/build.sc
index 826e01ea..7dae6dde 100755
--- a/build.sc
+++ b/build.sc
@@ -1,12 +1,10 @@
 import $file.ci.shared
 import $file.ci.upload
 import $ivy.`org.scalaj::scalaj-http:2.4.2`
-import $ivy.`de.tototec::de.tobiasroeser.mill.vcs.version_mill0.9:0.1.2`
-import $ivy.`com.github.lolgab::mill-mima_mill0.9:0.0.4`
+import $ivy.`de.tototec::de.tobiasroeser.mill.vcs.version::0.1.2`
 import $ivy.`net.sourceforge.htmlcleaner:htmlcleaner:2.24`
 import java.nio.file.attribute.PosixFilePermission
 
-import com.github.lolgab.mill.mima.Mima
 import coursier.maven.MavenRepository
 import de.tobiasroeser.mill.vcs.version.VcsVersion
 import mill._
@@ -146,14 +144,12 @@ trait MillCoursierModule extends CoursierModule {
 trait MillApiModule
     extends MillPublishModule
     with ScalaModule
-    with MillCoursierModule
-    with Mima {
+    with MillCoursierModule {
   def scalaVersion = Deps.scalaVersion
   override def ammoniteVersion = Deps.ammonite.dep.version
 //  def compileIvyDeps = Agg(Deps.acyclic)
 //  def scalacOptions = Seq("-P:acyclic:force")
 //  def scalacPluginIvyDeps = Agg(Deps.acyclic)
-  override def mimaPreviousVersions: T[Seq[String]] = Seq("0.10.0-M2", millLastTag()).distinct
 }
 
 trait MillModule extends MillApiModule { outer =>
